great_expectations/checkpoint/util.py

```diff
@@ -443,11 +442,22 @@ def get_updated_action_list(
     return list(base_action_list_dict.values())
 
 
+def batch_request_in_validations(
```

Comment:
suggested rename to: `does_validation_have_batch_request`; boolean check methods should be in the form of a question

Reply:
Let's put this in a relevant module or class and not add more to our util files (that should be removed).

Reply:
@donaldheppner Fixed (and using `does_batch_request_in_validations_contain_batch_data()` as well for the runtime data containing `DataFrame`).  Thanks.